__tests__/app.test.js

```diff
@@ -265,6 +265,42 @@ describe("App GET", () => {
         });
     });
   });
+  describe("GET /api/users", () => {
+    test("should return array containing all user objects", () => {
+      return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then((response) => {
+          const body = response.body;
+          expect(Array.isArray(body.users)).toBe(true);
```

Comment:
No need to check if it is an array, so we should have line 276 on the next test and delete this one 🙂